### ISSUE TITLE
Controls in table format

### ISSUE THREAD

**Issue by spbreed** [OWASP/www-project-ai-security-and-privacy-guide#35]
Do we have risks and controls mapping described in table format?

**Comment by robvanderveer** [comment]
Yes we do, it's called the matrix. Have you found it in the document? That's what you're looking for?

**Comment by spbreed** [comment]
You mean this section? https://github.com/OWASP/www-project-ai-security-and-privacy-guide/blob/main/content/ai_exchange/content/docs/ai_security_overview.md#ai-security-matrix

I am looking for something in md table format, something like this. https://github.com/OWASP/www-project-ai-security-and-privacy-guide/blob/main/content/ai_exchange/content/docs/ai_security_overview.md#how-about-generative-ai-eg-llm

**Comment by robvanderveer** [comment]
What would be in the columns and what would be in the rows of this table that you seek?

**Comment by robvanderveer** [comment]
Will close this issue Mar 7, assuming there is no issue anymore given the lack of response
